packages/design-system-react-native/package.json

```diff
@@ -47,23 +47,20 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "dependencies": {
-    "@metamask/design-system-twrnc-preset": "workspace:^",
-    "@metamask/design-tokens": "workspace:^",
-    "@metamask/utils": "^11.4.0",
-    "@solana/addresses": "^2.1.0",
-    "@testing-library/react-hooks": "^8.0.1",
-    "bitcoin-address-validation": "^3.0.0",
-    "react-native-gesture-handler": "^1.10.3",
-    "react-native-jazzicon": "^0.1.2",
-    "react-native-reanimated": "3.3.0"
+    "react-native-jazzicon": "^0.1.2"
   },
   "devDependencies": {
     "@babel/core": "^7.23.5",
     "@babel/preset-env": "^7.26.0",
     "@babel/preset-react": "^7.25.9",
     "@babel/preset-typescript": "^7.23.3",
     "@metamask/auto-changelog": "^5.0.2",
+    "@metamask/design-system-twrnc-preset": "workspace:^",
+    "@metamask/design-tokens": "workspace:^",
+    "@metamask/utils": "^11.0.0",
+    "@solana/addresses": "^2.0.0",
```

Comment:
We are pinning `@solana/addresses` to major version `^2.0.0` to maintain compatibility with MetaMask Mobile, which uses `2.0.0` (see [metamask-mobile/package.json#L280](https://github.com/MetaMask/metamask-mobile/blob/main/package.json#L280)).